src/currentprocess/homethunk.rs

```diff
@@ -19,3 +33,15 @@ impl home::Env for Box<dyn CurrentProcess + 'static> {
         VarSource::var_os(self.deref(), key)
     }
 }
+
+impl home::Env for OSProcess {
+    fn home_dir(&self) -> Option<PathBuf> {
+        home::OS_ENV.home_dir()
```

Comment:
If we use default implementation for `home::Env`, we don't need `home::OS_ENV` here.
cc https://github.com/brson/home/pull/23/files#r449375855